apps/runtime-demo/3005-runtime-host/webpack.config.js

```diff
@@ -1,13 +1,57 @@
+const path = require('path');
 const { registerPluginTSTranspiler } = require('nx/src/utils/nx-plugin.js');
 registerPluginTSTranspiler();
-
+const { ModuleFederationPlugin } = require('@module-federation/enhanced');
 const { composePlugins, withNx } = require('@nx/webpack');
 const { withReact } = require('@nx/react');
 
-module.exports = composePlugins(withNx(), withReact(), (config) => {
+module.exports = composePlugins(withNx(), withReact(), (config, context) => {
+  // const webpack = require('/Users/bytedance/outter/universe/node_modules/.pnpm/webpack@5.89.0_@swc+core@1.3.99_esbuild@0.19.7/node_modules/webpack/lib/index.js')
+  // const ModuleFederationPlugin = webpack.container.ModuleFederationPlugin;
+  config.plugins.push(
+    new ModuleFederationPlugin({
+      name: 'runtime_host',
+      remotes: {
+        // remote2: 'runtime_remote2@http://localhost:3007/remoteEntry.js',
+        remote1: 'runtime_remote1@http://localhost:3006/remoteEntry.js',
+      },
+      // library: { type: 'var', name: 'runtime_remote' },
+      filename: 'remoteEntry.js',
+      exposes: {
+        './Button': './src/Button.tsx',
+      },
+      shared: {
+        lodash: {},
+        antd: {},
+        react: {},
+        'react/': {},
+        'react-dom': {},
+        'react-dom/': {},
+      },
+      // runtimePlugins: [path.join(__dirname, './runtimePlugin.ts')],
```

Comment:
Whether this can be opened, and then supplement the set test of this

Reply:
yes, i will remove the comment